### PR TITLE
update plan replayer continuous capture feature

### PR DESCRIPTION
**Pull request by Yisaer**
### Which TiDB version(s) do your changes apply to? (Required)

update plan replayer continuous capture feature

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-plan-replayer.md [hunk 1]
```diff
@@ -242,3 +242,25 @@ mysql> SELECT * FROM mysql.plan_replayer_status;
 > **注意：**
 > 
 > `PLAN REPLAYER CAPTURE` 的结果文件最多会在 TiDB 集群中保存一个小时，超时后 TiDB 会将其删除。
+
+### 使用 `PLAN REPLAYER CONTINUOUS CAPTURE`
+
+`PLAN REPLAYER CONTINUOUS CAPTURE` 功能通过系统变量 [`tidb_enable_plan_replayer_continuous_capture`](/system-variables.md#tidb_enable_plan_replayer_continuous_capture) 控制。要开启 `PLAN REPLAYER CONTINUOUS CAPTURE`，将变量值设为 `ON`。
+
+开启 `PLAN REPLAYER CONTINUOUS CAPTURE` 功能后， TiDB 将根据 SQL DIGEST 和 PLAN DIGEST 为维度异步地将业务 SQL 以 PLAN REPLAYER 的方式进行记录，对于相同 DIGEST 的 SQL 与 PLAN, `PLAN REPLAYER CONTINUOUS CAPTURE` 不会进行重复记录。
```
Comment (TomShawn) [inline sql-plan-replayer.md:250]: 能否说明一下 PLAN REPLAYER CONTINUOUS CAPTURE 的用户使用场景？
```suggestion
开启 `PLAN REPLAYER CONTINUOUS CAPTURE` 功能后，TiDB 将以 SQL DIGEST 和 PLAN DIGEST 为维度异步地将业务 SQL 语句以 `PLAN REPLAYER` 的方式进行记录，对于相同 DIGEST 的 SQL 语句与执行计划, `PLAN REPLAYER CONTINUOUS CAPTURE` 不会重复记录。

`PLAN REPLAYER CONTINUOUS CAPTURE` 功能通过系统变量 [`tidb_enable_plan_replayer_continuous_capture`](/system-variables.md#tidb_enable_plan_replayer_continuous_capture) 控制。要开启 `PLAN REPLAYER CONTINUOUS CAPTURE`，将变量值设为 `ON`。
```

File: sql-plan-replayer.md [hunk 2]
```diff
@@ -242,3 +242,25 @@ mysql> SELECT * FROM mysql.plan_replayer_status;
...
+
+### 查看 `PLAN REPLAYER CONTINUOUS CAPTURE` 抓取结果
+
+同 `PLAN REPLAYER CAPTURE`
```
Comment (TomShawn) [inline sql-plan-replayer.md:254]:
```suggestion
查看 `PLAN REPLAYER CONTINUOUS CAPTURE` 抓取结果的方法同[查看 `PLAN REPLAYER CAPTURE` 抓取结果](#查看-plan-replayer-continuous-capture-抓取结果)
```

File: sql-plan-replayer.md [hunk 3]
```diff
@@ -242,3 +242,25 @@ mysql> SELECT * FROM mysql.plan_replayer_status;
...
+
+开启 `PLAN REPLAYER CONTINUOUS CAPTURE` 功能后， TiDB 将根据 SQL DIGEST 和 PLAN DIGEST 为维度异步地将业务 SQL 以 PLAN REPLAYER 的方式进行记录，对于相同 DIGEST 的 SQL 与 PLAN, `PLAN REPLAYER CONTINUOUS CAPTURE` 不会进行重复记录。
+
+### 查看 `PLAN REPLAYER CONTINUOUS CAPTURE` 抓取结果
+
+同 `PLAN REPLAYER CAPTURE` 
+
+## 使用 EXTRACT HTTP 接口导出负载计划
+
+EXTRACT HTTP 的导出接口如下。
```
Comment (TomShawn) [inline sql-plan-replayer.md:258]: 能否说一下用户使用场景？

**Comment by ran-huang** [comment]
@time-and-fate Could you please address the comments?

File: sql-plan-replayer.md [hunk 4]
```diff
@@ -242,3 +242,25 @@ mysql> SELECT * FROM mysql.plan_replayer_status;
 > **注意：**
 > 
 > `PLAN REPLAYER CAPTURE` 的结果文件最多会在 TiDB 集群中保存一个小时，超时后 TiDB 会将其删除。
+
+### 使用 `PLAN REPLAYER CONTINUOUS CAPTURE`
+
+开启 `PLAN REPLAYER CONTINUOUS CAPTURE` 功能后，TiDB 将以 SQL DIGEST 和 PLAN DIGEST 为维度异步地将业务 SQL 语句以 `PLAN REPLAYER` 的方式进行记录，对于相同 DIGEST 的 SQL 语句与执行计划, `PLAN REPLAYER CONTINUOUS CAPTURE` 不会重复记录。
```
Comment (ran-huang) [inline sql-plan-replayer.md:248]:
```suggestion
开启 `PLAN REPLAYER CONTINUOUS CAPTURE` 功能后，TiDB 将以 SQL DIGEST 和 PLAN DIGEST 为维度异步地将业务 SQL 语句以 `PLAN REPLAYER` 的方式进行记录，对于相同 DIGEST 的 SQL 语句与执行计划，`PLAN REPLAYER CONTINUOUS CAPTURE` 不会重复记录。
```

File: sql-plan-replayer.md [hunk 5]
```diff
@@ -242,3 +242,25 @@ mysql> SELECT * FROM mysql.plan_replayer_status;
...
+
+`PLAN REPLAYER CONTINUOUS CAPTURE` 功能通过系统变量 [`tidb_enable_plan_replayer_continuous_capture`](/system-variables.md#tidb_enable_plan_replayer_continuous_capture) 控制。要开启 `PLAN REPLAYER CONTINUOUS CAPTURE`，将变量值设为 `ON`。
+
+### 查看 `PLAN REPLAYER CONTINUOUS CAPTURE` 抓取结果
+
+查看 `PLAN REPLAYER CONTINUOUS CAPTURE` 抓取结果的方法同[查看 `PLAN REPLAYER CAPTURE` 抓取结果](#查看-plan-replayer-continuous-capture-抓取结果)。
+
+## 使用 EXTRACT HTTP 接口导出负载计划
+
+EXTRACT HTTP 的导出接口如下。
```
Comment (ran-huang) [inline sql-plan-replayer.md:258]: 能否简单介绍下这个功能的价值，比如什么场景下用户需要导出负载计划？

File: sql-plan-replayer.md [hunk 6]
```diff
@@ -242,3 +242,25 @@ mysql> SELECT * FROM mysql.plan_replayer_status;
...
+
++ 通过以下接口可以获取开始时间 `${begin}` 和结束时间 `${end}` 的 ZIP 格式的负载计划：
+
+    {{< copyable "" >}}
```
Comment (ran-huang) [inline sql-plan-replayer.md:262]:
```suggestion
```

File: system-variables.md [hunk 7]
```diff
@@ -1541,16 +1541,13 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`ON`
 - 这个变量用来控制是否开启 [`PLAN REPLAYER CAPTURE` 功能](/sql-plan-replayer.md#使用-plan-replayer-capture-抓取目标计划)。默认值 `ON` 代表开启 `PLAN REPLAYER CAPTURE` 功能。
 
-### `tidb_enable_plan_replayer_continues_capture`
-
-> **警告：**
->
-> 当前版本中该变量控制的功能尚未完全生效，请保留默认值。
+### `tidb_enable_plan_replayer_continuous_capture`
```
Comment (ran-huang) [inline system-variables.md:1544]:
```suggestion
### `tidb_enable_plan_replayer_continuous_capture` <span class="version-mark">从 v7.0.0 版本开始引入</span>
```

File: sql-plan-replayer.md [hunk 8]
```diff
@@ -242,3 +242,25 @@ mysql> SELECT * FROM mysql.plan_replayer_status;
 > **注意：**
 > 
 > `PLAN REPLAYER CAPTURE` 的结果文件最多会在 TiDB 集群中保存一个小时，超时后 TiDB 会将其删除。
+
+### 使用 `PLAN REPLAYER CONTINUOUS CAPTURE`
+
+开启 `PLAN REPLAYER CONTINUOUS CAPTURE` 功能后，TiDB 将以 SQL DIGEST 和 PLAN DIGEST 为维度异步地将业务 SQL 语句以 `PLAN REPLAYER` 的方式进行记录，对于相同 DIGEST 的 SQL 语句与执行计划, `PLAN REPLAYER CONTINUOUS CAPTURE` 不会重复记录。
+
+`PLAN REPLAYER CONTINUOUS CAPTURE` 功能通过系统变量 [`tidb_enable_plan_replayer_continuous_capture`](/system-variables.md#tidb_enable_plan_replayer_continuous_capture) 控制。要开启 `PLAN REPLAYER CONTINUOUS CAPTURE`，将变量值设为 `ON`。
```
Comment (ran-huang) [inline sql-plan-replayer.md:250]:
```suggestion
`PLAN REPLAYER CONTINUOUS CAPTURE` 功能通过系统变量 [`tidb_enable_plan_replayer_continuous_capture`](/system-variables.md#tidb_enable_plan_replayer_continuous_capture-从-v700-版本开始引入) 控制。要开启 `PLAN REPLAYER CONTINUOUS CAPTURE`，将变量值设为 `ON`。
```

File: sql-plan-replayer.md (hunk 5 above)
Comment (time-and-fate) [reply sql-plan-replayer.md]: 确认了一下，这个不是完全属于 plan replayer continuous capture 的功能描述，而且它需要依赖 stmt summary 的一个未 GA 功能。暂时先把这段删掉了。

**Review by ran-huang — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

File: system-variables.md [hunk 9]
```diff
@@ -1541,16 +1541,13 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`ON`
 - 这个变量用来控制是否开启 [`PLAN REPLAYER CAPTURE` 功能](/sql-plan-replayer.md#使用-plan-replayer-capture-抓取目标计划)。默认值 `ON` 代表开启 `PLAN REPLAYER CAPTURE` 功能。
 
-### `tidb_enable_plan_replayer_continues_capture`
-
-> **警告：**
->
-> 当前版本中该变量控制的功能尚未完全生效，请保留默认值。
+### `tidb_enable_plan_replayer_continuous_capture` <span class="version-mark">从 v7.0.0 版本开始引入</span>
```
Comment (ran-huang) [inline system-variables.md:1544]: added to 7.0.0 release notes in https://github.com/pingcap/docs-cn/pull/13312/files#r1148817483

**Pull request merged**